data/remnant/remnant.txt

```diff
@@ -857,3 +858,123 @@ outfitter "Remnant Salvage"
 	"Steering (Asteroid Class)"
 	"Steering (Planetary Class)"
 	"Korath Fire-Lance"
+
```

Comment:
These should be in `commodities.txt`.

Reply:
@Zitchas I have no view on where these should go. There don't appear to be faction specific commodities in commodities.txt. I checked the Coalition, Hai and Wanderer directories and I don't see any custom commodities defined, so it looks like we are setting precedent. Let me know if you wish me to move them.

Reply:
My initial thought is to move it to commodities.txt as suggested. My second is to create a separate `remnant commodities.txt`, but then my third is to leave these here with the rest of the government stuff. Which is all the options. lol.

Anyway, I think my preference would probably be the stand-alone `remnant commodities.txt` in the Remnant folder. That keeps them separate for ease of use; but also keeps them collected with the other Remnant stuff.

Reply:
I've split the commodities out into a "remnant commodities.txt" file. I'm currently playing through the remnant story line to test that everything comes on-line as expected. I've seen no issues with the first three job sets.